pkg/securitycontext/util.go

```diff
@@ -156,11 +187,117 @@ func securityContextFromPodSecurityContext(pod *v1.Pod) *v1.SecurityContext {
 		*synthesized.RunAsUser = *pod.Spec.SecurityContext.RunAsUser
 	}
 
+	if pod.Spec.SecurityContext.RunAsGroup != nil {
+		synthesized.RunAsGroup = new(int64)
+		*synthesized.RunAsGroup = *pod.Spec.SecurityContext.RunAsGroup
+	}
+
 	if pod.Spec.SecurityContext.RunAsNonRoot != nil {
 		synthesized.RunAsNonRoot = new(bool)
 		*synthesized.RunAsNonRoot = *pod.Spec.SecurityContext.RunAsNonRoot
 	}
 
+	if pod.Spec.SecurityContext.RunAsNonRootGroup != nil {
+		synthesized.RunAsNonRootGroup = new(bool)
+		*synthesized.RunAsNonRootGroup = *pod.Spec.SecurityContext.RunAsNonRootGroup
+	}
+
+	return synthesized
+}
+
+// TODO: remove the duplicate code
```

Comment:
I would be fine with you doing this TODO in an immediate follow-up, if you don't want to do it now.

Reply:
I don't see where this method is used - am I missing something?

Reply:
removed it